Packs/CommonTypes/IncidentFields/incidentfield-ExtraHop_Detection_ID.json

```diff
@@ -24,7 +24,8 @@
  "group": 0,
  "hidden": false,
  "associatedTypes": [
-  "ExtraHop Detection"
```

Comment:
This one is related to the ExtraHop integration, I agree it shouldn't be in the CommonTypes pack.
Please see if there's a different field you can use for Vectra Detection ID or you may create a new field in your pack.

Reply:
Hi @idovandijk , indeed I saw that this field related to ExtraHop but being part of the CommonTypes, I was initialy thinking of updating the description to be more generic but also the .json file name to be usable by everyone, not only ExtraHop as quite confusing.
I'll see want I can use instead.

Reply:
Yes, let's not touch the existing field as its name already suggests it's for another integration :)

Reply:
Hi @idovandijk , so far, the “ExtraHop” usage is only visible under the `description`, neither on the `name` or `id` or `cliName` directly. I’ve put aside the `associatedTypes` as this one can have multiples values for any fields.
This is only visible on the .json file name.
Are you OK if I simply update the description to be more generic ? So it could be used by any other integration.
Also should I change the file name (with a git move) or should I left it intact ?
Bonus question: Should I do the same changes on the two others "ExtraHop" incident fields `Detection Update Time`and `Detection End Time` ?
Thx in advance